pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -453,6 +453,10 @@ type DatacenterSpec struct {
 	// ignoring cluster-specific settings.
 	EnforceAuditLogging bool `json:"enforceAuditLogging,omitempty"`
 
+	// Optional: EnableAuditWebhook allows admins to control webhook backend for audit logs of all the clusters within the DC,
+	// ignoring cluster-specific settings.
+	EnableAuditWebhook *AuditWebhookBackendSettings `json:"enableAuditWebhook,omitempty"`
```

Comment:
This field name still is a boolean expression (something I would expect to set like this: `enableAuditWebhook: true`), please give it a more descriptive name, e.g. `enforcedAuditWebhookSettings`.